avalanche/training/strategies/icarl.py

```diff
@@ -130,9 +130,11 @@ def before_training_exp(self, strategy: 'BaseStrategy', **kwargs):
         tid = strategy.clock.train_exp_counter
         benchmark = strategy.experience.benchmark
         nb_cl = benchmark.n_classes_per_exp[tid]
+        previous_seen_classes = sum(benchmark.n_classes_per_exp[:tid])
```

Comment:
shouldn't this be a set?

Reply:
no, it's just a counter that counts how many classes I've already observed. Maybe the name is misleading, I can change it.

Reply:
Does the strategy works if you have Exp.1 with classes [2, 3], and Exp.2 with classes [0, 1]? If so, you can ignore my comment.